playbooks/install_operators.yml

```diff
@@ -0,0 +1,45 @@
+# A playbook to create a catalog source and install some of its operators
+# For disconnected environments, it is required that the IDMS/ICSP are applied in advance
+# Please update the <placeholder> variables according your environment before running
+---
+- name: Install Operators
+  hosts: localhost
+  vars:
+    source_ns: "openshift-marketplace"
+    catalog_name: "<my_catalog>"
+    catalog_image: "<local_registry>:<local_registry_port>/<org>/<catalog>:latest"
+  tasks:
+
+    - name: Create a CatalogSource
+      ansible.builtin.include_role:
+        name: redhatci.ocp.catalog_source
+      vars:
+        cs_name: "{{ catalog_name }}"
+        cs_namespace: "{{ source_ns }}"
+        cs_image: "{{ catalog_image }}"
+        cs_publisher: "The publisher"
+
+    - name: Install OCS Operator
```

Comment:
The problem I see with this playbook is that is very specific, it will only install OCS and ACM. I think it would be more useful being more generic, wdyt?